packages/terra-menu/src/Menu.jsx

```diff
@@ -199,6 +199,7 @@ class Menu extends React.Component {
         hookshotPostionFixed
         isHeaderDisabled
         isContentFocusDisabled
+        popupContentRole={null}
```

Comment:
Why do we have to set the property value to null?  Won't it be undefined just because the property is not specified?

Reply:
Added since popup currently defaults to role as dialog: https://github.com/cerner/terra-framework/blob/main/packages/terra-popup/src/Popup.jsx#L123